### PR TITLE
Create templatized debian files

### PR DESCRIPTION
**Pull request by zyoung51**
Create Quanta-D51 and Quanta-T41 definitions

### CODE REVIEW

**Comment by zyoung51** [comment]
@RackHD/corecommitters @tannoa2 @keedya @uppalk1

**Comment by jlongever** [comment]
+1

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+ - ./build-package.bash quanta-t41 "${TRAVIS_BRANCH}"
+ - if [ ! -e deb ]; then mkdir deb && cp -a *.deb deb/; fi
+
+deploy:
+  - provider: bintray
+    file: .bintray-deb.json
+    user: $BINTRAY_USER
+    key: $BINTRAY_KEY
+    on:
+      branch: master
```
Comment (heckj) [inline .travis.yml:23]: Am I getting this right that it'll try to build every time, but only upload the packages on merge to master?

File: debianstatic/postinst.in [hunk 2]
```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+FILE=/var/renasar/on-http/.skupack/{{name}}.tar.gz
+SKUFILE=/var/renasar/on-http/.skupack/{{name}}.sku
+CODE=`curl -s -X POST --data-binary @${FILE} -o ${SKUFILE} -w '%{http_code}' localhost:8080/api/common/skus/pack`
```
Comment (heckj) [inline debianstatic/postinst.in:4]: this is leveraging the idea that rackhd will be available locally at package install time, but the upstarts of those scripts are set up explicitly to _not_ start the services on install. Lines like https://github.com/RackHD/on-tftp/blob/master/debian/on-tftp.upstart#L10 in each of the application packages were added to make sure that the services wouldn't auto-start unless a specific file exists (i.e. gotta touch a file first) for each application.

I think the end result here is that on a stock, straight from "deb" install is processing, the registration will fail, requiring the user to manually re-register

Comment (zyoung51) [inline debianstatic/postinst.in:4]: Yep, that's what happens if the API is offline.  The API has to be up to service the request to create the SKU from the rules definition and process the nodes against it.

The else prompts the user to retry registering:
-  echo SKU Pack registration failed. Please retry registering
  -  echo the pack at: ${FILE}

Also note that northbound/southbound API separation, authentication, and port shifting will either fully break this strategy or at least require parsing the .conf file to find the correct settings to use.  An assumption is also being made that this is being run on the entity hosting on-http.

**Comment by heckj** [comment]
:+1: Some possible questions about the postinst pieces for registering, but good regardless

**Comment by jlongever** [comment]
@heckj re postinst:  Zack and I discussed this earlier too,  I made similar comments.  I think the post "install" meaning is to really "register" the skupack.  The compromise to my complaining was to also deploy raw tarballs to bintray as well,  giving a user the option to pull them down into a local repo and register manually (or automatically through some infra mgmt piece).

**Comment by jlongever** [comment]
@RackHD/corecommitters is this OK to merge?

**Pull request merged**